app/src/main/java/de/westnordost/streetcomplete/quests/wheelchair_access/AddWheelchairAccessBusiness.java

```diff
@@ -36,13 +41,25 @@
 				"courthouse", "embassy", "car_wash", "car_rental",
 				"marketplace", "fuel", "driving_school", "dentist",
 				"doctors", "clinic", "pharmacy", "veterinary",
-				"place_of_worship", "townhall"};
+				"place_of_worship", "townhall", "school", "kindergarten",
+				"shelter", "hospital", "police", "community_centre",
+				"social_facility", "public_building", "university", "college",
+				"theatre", "arts_centre", "bbq", "nursing_home",
+				"ferry_terminal", "childcare", "shower", "prison", "studio",
+				"casino", "internet_cafe", "public_bookcase", "brothel",
+				"dojo", "gambling", "payment_terminal", "events_venue",
+				"public_bath", "training", "social_centre", "animal_breeding",
+				"animal_shelter", "retirement_home", "monastery", "sauna",
```

Comment:
retirement_home would be a spam candidate, too.

Reply:
Thanks, I'll remove this like public_building.